internal/latestrelease/finder.go

```diff
@@ -126,7 +126,7 @@ func (f *finder) Find() (releaseInfo *ReleaseInformation, err error) {
 	}
 
 	releaseInfo, newestVersion, err := f.loadOrGet()
-	if err != nil || releaseInfo == nil || svCurrentVersion.Compare(newestVersion) > 0 {
+	if err != nil || releaseInfo == nil || svCurrentVersion.Compare(newestVersion) >= 0 {
```

Comment:
Could you check why we don't get this error with mongocli v1.23.1?

```
/Downloads/mongocli_1.23.1_macos_x86_64/bin ./mongocli iam projects list
ID                         NAME
5e4e593f70dfbf1010295836   CLI ANDREA
61dc598fd1f07c7b37ba425d   CLI Bianca
5e429f2e06822c6eac4d59c9   CLI tests
607caaa2b4c1fd299fb542be   Migration
613625603e005358482462d9   ProjectFoobar
61279659d7d1a561be8ab56e   TF GUS
```

Reply:
I have downloaded the 1.23.1 binary from here https://github.com/mongodb/mongodb-atlas-cli/releases/tag/mongocli%2Fv1.23.1

Reply:
you are running against master. This issue may have been introduced with one of your latest changes that are not in the 1.23.1

Reply:
Thanks, precisely. atlascli has my change whereas mcli didn't have. The bug was introduced with https://github.com/mongodb/mongodb-atlas-cli/pull/1010/files in the comparison. 

It was changed when I updated the logic which. was 
```
	if currentVersion.Compare(v) < 0 {
		return true, &version.ReleaseInformation{
			Version:     v.Original(),
			PublishedAt: release.GetPublishedAt().Time,
		}, nil
	}
``
`